Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -97,7 +97,9 @@ struct FPTIBatchData: Codable {
         let merchantAppVersion: String = Bundle.main.infoDictionary?[kCFBundleVersionKey as String] as? String ?? "N/A"
 
         let merchantID: String?
-        
+
+        let payPalContextID: String?
```

Comment:
Take it or leave it comment - adding a docstring to this property with some of the description for the different values this can represent; (ecToken, orderID, etc).

Reply:
Might help a new person understanding what this represents, or future us

Reply:
Good call! I also added a docstring to each of the clients describing what the value is for that particular client: https://github.com/braintree/braintree_ios/pull/1181/commits/e7c3fe0b513a861d6899cbab6aeee05ba077b3ae